src/main/java/net/datafaker/transformations/JavaObjectTransformer.java

```diff
@@ -138,6 +139,11 @@ public Stream<Object> generateStream(final Schema<Object, ?> schema, long limit)
                 .limit(limit);
     }
 
+    @Override
+    public void writeToOutputStream(OutputStream outputStream, Schema<Object, ?> schema, long limit) {
+        throw new UnsupportedOperationException("Not implemented yet");
```

Comment:
not sure how it should look like in this case

Reply:
I think this is fine for now.

Maybe add some JavaDoc as well.

Reply:
added a comment for that